backend/src/main/resources/db/migration/V1__init.sql

```diff
@@ -3,10 +3,8 @@ create table subscriptions_table (
     name text not null,
     interval VARCHAR(255) not null,
     active boolean not null,
-    conditions_met boolean not null,
```

Comment:
Just a comment: When we do it this, we need to call in the frontend the `/evaluateSubscription` endpoint for each subscription the user has on the `/subscriptions` frontend page. Is this what we want to do? Should the user be shown the current values, or the stored values of the trigger with an additional button to evaluate the triggers?

Reply:
As discussed: We can still add it again later. Latest when we implement notifications via some channel, we probably need to store the last result anyway.